compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/JvmValueGen.java

```diff
@@ -317,20 +317,20 @@ private void createLambdas(ClassWriter cw, LambdaMetadata lambdaGenMetadata) {
         }
     }
 
-    private void createObjectFields(ClassWriter cw, List<BField> fields) {
+        private void createObjectFields(ClassWriter cw, Map<String, BField> fields) {
```

Comment:
Formatting issue?

Reply:
Seems to be the case with multiple changes in this class.

Reply:
Ah damn.. There were a lot of formatting changes when resolving a conflict. I think these were introduced when resolving them.